### PR TITLE
AGDIGGER-65 - jenkins2

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Add support for jenkins2 image

ping @aliok

### CODE REVIEW

File: docker/jenkins1-centos/plugins.txt [hunk 1]
```diff
@@ -1 +1 @@
-script-realm:1.5
\ No newline at end of file
+cloudbees-folder:5.13
```
Comment (aliok) [inline docker/jenkins1-centos/plugins.txt]: @wtrocki Could you give a short explanation of what this is?

**Comment by aliok** [comment]
Looks good

**Comment by wtrocki** [comment]
Image pushed to dockerhub. Verification performed on shared cluster.

**Comment by wtrocki** [comment]
Added image as default and updated dockerhub

**Pull request merged**